src/App.tsx

```diff
@@ -19,55 +51,52 @@ export const App: React.FC = () => {
               placeholder="Enter a part of the name"
               className="input"
               data-cy="search-input"
+              value={query}
+              onChange={handleQueryChange}
             />
           </div>
 
-          <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
-            <div className="dropdown-content">
-              <div className="dropdown-item" data-cy="suggestion-item">
+          {/* <div className="dropdown-item" data-cy="suggestion-item">
                 <p className="has-text-link">Pieter Haverbeke</p>
-              </div>
+          </div> */}
 
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Bernard Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
+          {!chosenPerson && suggestions.length > 0 && (
+            <div
+              className="dropdown-menu"
+              role="menu"
+              data-cy="suggestions-list"
+            >
+              <div className="dropdown-content">
+                {suggestions.map(suggestion => (
+                  <div
+                    key={suggestion.name}
```

Comment:
You are generating keys for your list items using the 'name' property of each person. This is not recommended as names can be non-unique in some cases. It's better to use unique ids for keys. You can refer to [this article](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052) for more information on why it's important to use unique keys in React.